docs-source/usersguide/library/02_compiling.rst

```diff
@@ -115,36 +118,24 @@ Configure (press “c”) again.  Adjust any variables that cause an error.
 Continue to configure (press “c”) until no starred CMake variables are
 displayed, then press “g” to generate the makefiles for building the project.
 
-Step 2: Build and Install
-=========================
-
-Build and install OpenMM with the command::
-
-    make install
-
-If you are installing to a system area, such as /usr/local/openmm/, you will
-need to type::
-
-    sudo make install
-
-Step 3: Install the Python API
-==============================
-
-Build and install the Python API with the command::
-
-    make PythonInstall
+Step 2: Build
+=============
 
-If you are installing into the system Python, such as /usr/bin/python, you will
-need to type:
-::
+Build OpenMM with the command::
 
-    sudo make PythonInstall
+    make
 
 .. _test-your-build:
 
-Step 4: Test your build
+Step 3: Test your build
 =======================
 
+This step is optional but recommended. Tests can take up to several minutes depending on your
+hardware configuration.
```

Comment:
That's a bit of an underestimate!  When building all platforms, it can easily take half an hour or more to run all tests.